scanpointgenerator/generators/arraygenerator.py

```diff
@@ -0,0 +1,71 @@
+from scanpointgenerator.compat import np
+from scanpointgenerator.core import Generator
+
+
+@Generator.register_subclass("scanpointgenerator:generator/ArrayGenerator:1.0")
+class ArrayGenerator(Generator):
+    """Generate points fron a given list of positions"""
+
+    def __init__(self, names, units, points, alternate_direction=False):
```

Comment:
names -> name (str)
units (str)
points (list of float)
alternate_direction -> alternate (bool)